pom.xml

```diff
@@ -114,6 +114,7 @@
                                 <configuration>
                                     <sources>
                                         <source>${project.basedir}/src/main/301+-nondb/scala</source>
+                                        <source>${project.basedir}/src/main/shim301/scala</source>
```

Comment:
Nit: Shouldn't this just be `301` to follow the convention set by the other directories?  Similar comment for the other `shim*` directories that were added.

Reply:
I named them this way so reviewers can clearly distinguish between files that were moved from the shims module. These folders will go away and everything will be merged into the existing folders i.e. `301db`, `301until310-all` etc

Reply:
> These folders will go away

I still think these should be named appropriately.  It's 301db, not shim301db, for example.  Someone could decide to use these directories for a file unrelated to this shim refactoring work that requires compiling only on a very specific version.  If that happens then the folder will not go away, and we'll wonder why the naming was not consistent.

Reply:
gotcha! done